packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/AccountPanel/ContractNew/ContractBillingDetailsModal/EditContractModal.tsx

```diff
@@ -306,102 +346,110 @@ export const EditContractModal = ({
       <ModalOverlay />
       <ModalContent
         className='border-r-2 flex gap-6 bg-transparent shadow-none border-none'
-        style={{ minWidth: '971px', minHeight: '80%', boxShadow: 'none' }}
+        style={{ minWidth: '971px', minHeight: '80vh', boxShadow: 'none' }}
       >
-        <div
-          className='flex flex-col gap-4 px-6 pb-6 pt-4 bg-white h-auto rounded-lg justify-between relative'
-          style={{ width: '424px', maxWidth: '424px' }}
-        >
-          {billingDetailsFormOpen && (
-            <>
-              <div
-                className='h-[95%] rounded-l-lg left-[-12px] bg-white absolute w-[12px] shadow-inner'
-                style={{ boxShadow: 'inset -3px 1px 6px 0px #10182814' }}
-              />
-
-              <div className='flex flex-col relative'>
-                <ModalHeader className='p-0 text-lg font-semibold'>
-                  <div>
-                    {data?.contract?.organizationLegalName ||
-                      organizationName ||
-                      "Unnamed's "}{' '}
-                  </div>
-                  <span className='text-sm font-normal'>
-                    These details are required to issue invoices
-                  </span>
-                </ModalHeader>
-
-                <BillingDetailsForm
-                  values={addressState.values}
-                  formId={'billing-details-address-form'}
-                />
-              </div>
-              <ModalFooter className='p-0 flex'>
-                <Button
-                  variant='outline'
-                  colorScheme='gray'
-                  onClick={() => setBillingDetailsOpen(false)}
-                  className='w-full'
-                  size='md'
-                >
-                  Cancel
-                </Button>
-                <Button
-                  className='ml-3 w-full'
-                  size='md'
-                  variant='outline'
-                  colorScheme='primary'
-                  onClick={handleSaveAddressChanges}
-                >
-                  Confirm
-                </Button>
-              </ModalFooter>
-            </>
+        <motion.div
```

Comment:
Can we deprecate the use of framer-motion? I think we can achieve animations just with tailwind and some data attributes. We should also remove framer-motion since it's huge and it's pretty worthless just to use it for animating transitions. What do you think?

Reply:
I would keep it for now, it's not causing us any performance issue. We can always explore optimization opportunities, especially since it offers lightweight options, and (speaking for myself)  it has a benefit of better dev exp